File: Data Structures & Algorithms/max-water-container/submission-0.java
class Solution {
    public int maxArea(int[] heights) {
        int res = 0;
        // Testing brute force solution (n^2)
        for (int i = 0; i< heights.length; i++){
            for (int j= heights.length-1; j>0; j--){
                int water = Math.min(heights[i],heights[j]) * Math.abs(i-j);
                res = Math.max(res, water);
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/merge-strings-alternately/submission-0.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        p1 , p2 = 0 , 0
        minima = (min(len(word1), len(word2))) 
        res = ""

        while p1 < minima and p2 < minima:
            res += (word1[p1])
            p1 += 1
            res += (word2[p2])
            p2 += 1
        if p1:
            res += (word1[p1:])

        if p2:
            res += (word2[p2:])

        return res